1주차/2941.py

```diff
@@ -0,0 +1,12 @@
+#2941
+#크로아티아 영어 치환
+c=['c=','c-','dz=','d-','lj','nj','s=','z=']
+str=input()
+
+for i in range(len(c)):
+  if str.find(c[i])!=-1:
+    str=str.replace(c[i],'a')
```

Comment:
```suggestion
for i in range(len(c)):
    str=str.replace(c[i],'a')
```
replace는 바꿀게 없다고 에러가 뜬다거나 하지 않으니까 굳이 find로 먼저 확인해주지 않아도 돼요.
오히려 필요없는 확인작업이 한 번 늘어서 코드가 더 느려져요